pkg/podres/mock_PodresourceClient.go

```diff
@@ -0,0 +1,76 @@
+package podres
+
+import (
+	context "context"
+
+	grpc "google.golang.org/grpc"
+
+	mock "github.com/stretchr/testify/mock"
+	podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
```

Comment:
we should use (in general, not just here) v1 now

Reply:
AFAIK, we are replacing kubernetes with a customized version which only support v1alpha hence using that here